.travis.yml

```diff
@@ -8,26 +8,33 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
+
+matrix:
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
--prefer-lowest will get the lowest dependencies base on you composer configuration.

In this case, we can prevent using methods/tools from dependencies that older versions does not have.

Basically, it's will prevent from created issues like "Your library throw an undefined method error"  and discover it's because user has not enough higher dependencies, because composer will tell him before that.

Many project use this system. See Symfony for example: https://github.com/symfony/symfony/blob/dd7583d4851fa10f711b3f7e0727f80236b22f77/.travis.yml#L16-L17